### PR TITLE
bpo-47103: Copy pgort140.dll into output directory when building PGInstrument on Windows

### PR DESCRIPTION
**Pull request by zooba**
https://bugs.python.org/issue47103

### CODE REVIEW

**Comment by neonene** [comment]
The following 2 tests in test_embed.py failed with the same error:

test_init_pybuilddir_win32
test_init_pyvenv_cfg
```
Traceback (most recent call last):
  File "C:\cp\Lib\contextlib.py", line 155, in __exit__
    self.gen.throw(typ, value, traceback)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cp\Lib\test\test_embed.py", line 1225, in tmpdir_with_python
    yield tmpdir
    ^^^^^^^^^^^^
  File "C:\cp\Lib\test\test_embed.py", line 1347, in test_init_pybuilddir_win32
    self.check_all_configs("test_init_compat_config", config,
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cp\Lib\test\test_embed.py", line 738, in check_all_configs
    out, err = self.run_embedded_interpreter(testname,
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\cp\Lib\test\test_embed.py", line 117, in run_embedded_interpreter
    self.assertEqual(p.returncode, returncode,
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 3221225781 != 0 : bad returncode 3221225781, stderr is ''
```

File: PCbuild/python.vcxproj [hunk 1]
```diff
@@ -132,9 +132,6 @@ set PYTHONPATH=$(PySourcePath)Lib
   </Target>
   <Target Name="GeneratePythonBat" AfterTargets="AfterBuild">
     <PropertyGroup>
```
Comment (neonene) [inline PCbuild/python.vcxproj:135]:
```suggestion
    <PropertyGroup>
      <_PGOPath Condition="$(Configuration) == 'PGInstrument'">@set PATH=%PATH%%3B$(OutDir)</_PGOPath>
```

Comment (neonene) [reply PCbuild/python.vcxproj:135]: Leaving `$(_PGOPath)` like this fixes my errors. The way is more generic than  #32146.

Comment (zooba) [reply PCbuild/python.vcxproj:135]: No, your other PR is better. We shouldn't rely on PATH for resolving DLLs, but should copy the right one alongside the executable.

Comment (zooba) [reply PCbuild/python.vcxproj:135]: But I see you've closed it, so I'll add a similar change to this PR.

Comment (neonene) [reply PCbuild/python.vcxproj:135]: The additional change is perfect for PGO/non-PGO.
I can confirm PGInstrument builds work with this.

**Pull request merged**